cli/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/ray-project/kuberay/cli
+
+go 1.16
```

Comment:
operator and apiserver have been upgraded to 1.17. Can we upgrade Golang version in cli project as well?